operators/serviceflow-operator/task/TaskFabrik.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const assert = require('assert');
+const CONST = require('../../../common/constants');
+let ServiceInstanceUpdate, ServiceInstanceBackup, BlueprintTask;
+
+class TaskFabrik {
```

Comment:
'FabrikTask' would have been better name in my opinion.....

Reply:
This follows the existing Fabrik naming conventions ex. `JobFabrik`. Will keep it as is, Fabrik, aka factory generally sounds better as a suffix than a prefix. Naming again, subjective :-)